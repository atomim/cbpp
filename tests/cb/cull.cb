// a useless comment
dot 10, 10, 25, 25 // comment on a line
	' yeah
	// another indented comment
/*
	a comment block
*/
box 25,25,50,50,1
REMSTART
	another block comment
REMEND
print "YEAAAH!" // pls put on glasses
