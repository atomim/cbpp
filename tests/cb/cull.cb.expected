dot 10, 10, 25, 25 
box 25,25,50,50,1
print "YEAAAH!" 